packages/react/src/ActionList/Group.tsx

```diff
@@ -22,13 +21,12 @@ const Heading: React.FC<HeadingProps & React.HTMLAttributes<HTMLHeadingElement>>
   as: Component = 'h3',
   className,
   children,
-  sx = defaultSxProp,
```

Comment:
Anywhere where the props were spreading anyways like this `{...rest}` I removed the argument. I felt this was cleaner and will help us later when we drop the sx support (only needing to remove the SxProp type)